frontend/app/src/store/blockchain/index.ts

```diff
@@ -48,7 +48,7 @@ function getAccountBalance(account: BlockchainAccount, chainBalances: Blockchain
   const nativeAsset = account.nativeAsset;
   const balance = assets
     ? {
-        amount: assets[nativeAsset.toUpperCase()]?.amount ?? Zero,
+        amount: assets[nativeAsset]?.amount ?? Zero,
```

Comment:
Great. I remember doing the uppercase intentionally because there was something that was lowercase there initially. If all is good then let's merge. Thank you.